src/main/java/org/cp/confisual/plantuml/NevisAuthVisitor.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Author : AdNovum Informatik AG
+ */
+
+package org.cp.confisual.plantuml;
+
+import org.cp.confisual.nevisauth.AuthState;
+import org.cp.confisual.nevisauth.Domain;
+import org.cp.confisual.nevisauth.Entry;
+
+public interface NevisAuthVisitor {
```

Comment:
I think we can name this interface as 'NevisAuthConfigVisitor', move it to nevisauth package, we don't need a separate 'plantuml' package